components/device-mgt/org.wso2.carbon.device.mgt.core/src/test/java/org/wso2/carbon/device/mgt/core/permission/mgt/PermissionManagerServiceTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * you may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.device.mgt.core.permission.mgt;
+
+import org.apache.log4j.Logger;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.testng.Assert;
+import org.testng.IObjectFactory;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.ObjectFactory;
+import org.testng.annotations.Test;
+import org.wso2.carbon.device.mgt.common.permission.mgt.Permission;
+import org.wso2.carbon.device.mgt.common.permission.mgt.PermissionManagementException;
+import org.wso2.carbon.device.mgt.common.permission.mgt.PermissionManagerService;
+import org.wso2.carbon.registry.core.exceptions.RegistryException;
+
+import java.util.Properties;
+
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@PrepareForTest(PermissionUtils.class)
+public class PermissionManagerServiceTest {
+
+    private Permission permission;
+    private Logger log = Logger.getLogger(PermissionManagerServiceTest.class);
+    private String PERMISSION_URL = "permission/admin/device-mgt/test/testPermission";
+    private String PERMISSION_PATH = "permission/admin/device-mgt/test/testPermission";
+    private String PERMiSSION_METHOD = "ui.execute";
```

Comment:
PERMiSSION_METHOD needs to be PERMISSION_METHOD. Note 'I' not and not i. And also as above variables are constants you can use 'private static final'.